### DISCUSSION TITLE
vuepress编辑文章在vscode里面换行之后网页上并没有换行

### DISCUSSION THREAD

**Discussion by JIeJaitt** [vuepress-theme-hope/vuepress-theme-hope#3214] [Q&A] [answered]
## 同一段代码在vuepress-hope的显示效果是如下所示：

**直接就显示为一排了**
```text
1. 啊文档的发发呆
fasfafqwfawfg1大方
```
<img width="303" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/78fcd3ea-c4bb-495f-bdfe-58f3045d0014">

<img width="593" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/3295dede-3d2e-4b6c-8e4a-34754b1cbbc6">

**如果空开一行的话则会显示成这样**
```text
1. 啊文档的发发呆

fasfafqwfawfg1大方
```

<img width="224" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/38a24701-c162-4483-b174-892b1e1e2cf6">

<img width="309" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/c0ee44a2-322c-4d0d-9283-5802315c21fe">

## 如果是在hexo上面的话

**分开的话显示效果跟vuepress-hope的显示效果是一致的**

```text
1. 啊文档的发发呆

fasfafqwfawfg1大方
```

<img width="253" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/1d7ccd4d-7fc5-433b-a548-9d6a49d954ed">

<img width="306" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/3e9f702d-f6d5-4e7d-8674-7bfdaf67d59e">

```text
1. 啊文档的发发呆
fasfafqwfawfg1大方
```
**合并的话则会显示这种效果，有序列表下面的内容紧贴着有序列表。**

<img width="219" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/ba522176-8f96-4e52-b851-c4b5bc2d34fd">

<img width="573" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/659cac65-cc54-4bd1-90c0-023d2897cc93">

## 总结
vuepress和hexo在分开的时候显示是没区别的，但是合并在一起的时候，vuepress直接变成了一排，但是hexo是有序列表下面的内容会紧贴着列表下面，然后顺着这个我发现好像是不存在空行的
例如：
```text
1. 啊文档的发发呆

fasfafqwfawfg1大方

fasfafqwfawfg1大方
fasfafqwfawfg1大方
fasfafqwfawfg1大方
```
<img width="602" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/77219045/5c65037a-61a5-43ee-9988-c75389a613f4">

## 结果
就是说，我觉得hexo这种显示效果更好，hope大佬你说能不能向hexo这种显示效果优化一下？

**Comment by Mister-Hope** [comment, marked as answer]
This is the exact common mark spec. For common mark, see theme docs. We already provide you a markdown tutorial.

To switch between hard/soft breaks, toggle break(s) (not sure)  option of markdown-it, it defaults to `false` and `true` is the behaviour uou are describing
